src/Razor/src/Microsoft.VisualStudio.Editor.Razor/LspServerActivationTracker.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.Razor.Workspaces;
+
+namespace Microsoft.VisualStudio.Editor.Razor;
+
+[Shared]
+[Export(typeof(ILspServerActivationTracker))]
+internal class LspServerActivationTracker : ILspServerActivationTracker
+{
+    public bool IsActive { get; private set; }
```

Comment:
I got paid to write this service, because I am a professional developer. Years of training led to this moment!

Reply:
But did you ask AI to write it for you? 🤔